src/ScheduleService/Controllers/GroupsController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.ScheduleService.Business.Group.Interfaces;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Group;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Responses.Group;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ScheduleService.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class GroupsController
+{
+  [HttpPost]
+  public async Task<OperationResultResponse<CreateGroupResponse>> CreateAsync(
+    [FromServices] ICreateGroupCommand command,
+    [FromBody] CreateGroupRequest request)
+  {
+    return await command.ExecuteAsync(request);
+  }
+
+  [HttpGet]
```

Comment:
provide id in the route like this: [HttpGet("{groupId}")] and then use it as parameter - [FromRoute] Guid groupId
Same for edit/update/delete

Reply:
+